src/client.js

```diff
@@ -0,0 +1,97 @@
+"use strict";
+var W3CWebSocket = require('websocket').w3cwebsocket;
+class Client {
+  constructor(url, protocol) {
```

Comment:
protocol doesn't need to be an option, because it only changes if the code changes.
Maybe call it 'graphql-subscription' or 'graphql-subscription-protocol'